.cookiecutter/includes/docker-compose/services.yml

```diff
@@ -0,0 +1,18 @@
+nginx-proxy:
+  image: nginx:1.17.6-alpine
+  ports:
+    - '127.0.0.1:9083:9083'
+  environment:
+    - NGINX_SECURE_LINK_SECRET
+  volumes:
+    - ./conf/nginx/nginx.conf:/etc/nginx/nginx.conf:ro
+    - ./conf/nginx/includes/proxy.conf:/etc/nginx/includes/proxy.conf:ro
+    - ./conf/nginx/includes/errors.conf:/etc/nginx/includes/errors.conf:ro
+    - ./conf/nginx/includes/robots.conf:/etc/nginx/includes/robots.conf:ro
+    - ./conf/nginx/includes/abuse_response_headers.conf:/etc/nginx/includes/abuse_response_headers.conf:ro
+    - ./conf/nginx/includes/secure_links.conf:/etc/nginx/includes/secure_links.conf:ro
+    - ./conf/nginx/includes.dev/app_upstream.conf:/etc/nginx/includes/app_upstream.conf:ro
+    - ./conf/nginx/includes.dev/resolver.conf:/etc/nginx/includes/resolver.conf:ro
+    - ./conf/nginx/dev_host_bridge.sh:/etc/nginx/dev_host_bridge.sh:ro
+    - ./conf/nginx/envsubst.conf.template:/var/lib/hypothesis/nginx_envsubst.conf.template:ro
+  command: /bin/sh -c "/etc/nginx/dev_host_bridge.sh && envsubst '$${NGINX_SECURE_LINK_SECRET}' < /var/lib/hypothesis/nginx_envsubst.conf.template > /var/lib/hypothesis/nginx_envsubst.conf && exec nginx"
```

Comment:
Copied Via's existing NGINX service into this include so that `make template` includes it in `docker-compose.yml`.